configuration/enabling-desktop-icons/en.md

```diff
@@ -3,12 +3,34 @@ title = "Enabling Desktop Icons"
 +++
 # Enabling Desktop Icons
 
-You can enable desktop icons by opening Raven, clicking the cog wheel for Budgie Settings, then toggling Desktop Icons on under the Background section.
+You can enable or disable desktop icons easily depending on your personal preference.
+
+## Budgie
+
+With Budgie, start by opening Raven, either by clicking on its icon in the far right of the system tray or using the keyboard shortcut `Windows Key + A`.  Next, click the cog icon for Budgie Settings and then toggle "Desktop Icons" on under the Background section.
```

Comment:
Windows is a trademark so to refer to it as the windows key would involve marking it as a trademark. Should be referred to as the <kbd>Super</kbd> key